front/src/App.tsx

```diff
@@ -75,7 +77,8 @@ function DropappRedirect({ path }: DropappRedirectProps) {
   if (!user || !user["https://www.boxtribute.com/base_ids"])
     return <Navigate to={pathToRedirect} replace />;
 
-  const baseId = user["https://www.boxtribute.com/base_ids"][0];
+  const baseId =
+    globalPreferences.selectedBase?.id || user["https://www.boxtribute.com/base_ids"][0];
```

Comment:
If line 77 is true, this line would never be reached, right?

Reply:
For this to be reached, the `if` statement in line 77 should be negative, so this makes sense. @pylipp